apps/web/src/app/events/components/AttendanceCard/AttendanceDateInfo.tsx

```diff
@@ -4,6 +4,20 @@ import { formatDate } from "date-fns"
 import { nb } from "date-fns/locale"
 import React from "react"
 
+const getFormatString = (isInSameYear: boolean) => (isInSameYear ? "dd. MMMM" : "dd.MM.yyyy")
+
+const formatTime = (date: Date) => formatDate(date, "HH:mm", { locale: nb })
+
+const dateComponent = (label: string, dateStr: string, time: string) => (
+  <div>
+    <Text className="text-base">{label}</Text>
+    <div className="flex flex-row gap-2 text-slate-12 text-base sm:flex-col sm:gap-0 sm:text-sm">
+      <Text>{dateStr}</Text>
+      <Text>kl. {time}</Text>
+    </div>
+  </div>
+)
```

Comment:
Probably make this a real React component and just pass `<DateComponent />` at callsite